src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -443,6 +443,108 @@ hmi_apis::Common_Result::eType MessageHelper::MobileToHMIResult(
   return HMIResultFromString(result);
 }
 
+#ifdef SDL_REMOTE_CONTROL
+mobile_apis::DeviceRank::eType MessageHelper::StringToDeviceRank(
```

Comment:
DeviceRank is not used in RC baseline, please remove it. And add it later when needed.